### ISSUE TITLE
Tests: Hangs when generating Awesome theme after JSON Resume theme

### ISSUE THREAD

**Issue by peternowee** [hacksalot/HackMyResume#182]
The automated tests of `test-themes.js` seem to hang when generating the `awesome` theme after any of the `jsonresume-theme-...` themes:

      Testing themes against JANE-Q-FULLSTACKER resume (FRESH)
        ✓ HELLO-WORLD theme (FRESH) should generate without throwing an exception
        ✓ COMPACT theme (FRESH) should generate without throwing an exception (2275ms)
        ✓ MODERN theme (FRESH) should generate without throwing an exception (1645ms)
        ✓ UNDERSCORE theme (FRESH) should generate without throwing an exception
        ✓ AWESOME theme (FRESH) should generate without throwing an exception (4777ms)
        ✓ POSITIVE theme (FRESH) should generate without throwing an exception (1481ms)
        ✓ JSONRESUME-THEME-BOILERPLATE theme (FRESH) should generate without throwing an exception (516ms)
        ✓ JSONRESUME-THEME-SCEPTILE theme (FRESH) should generate without throwing an exception (1119ms)
        ✓ JSONRESUME-THEME-MODERN theme (FRESH) should generate without throwing an exception (394ms)
        ✓ JSONRESUME-THEME-CLASSY theme (FRESH) should generate without throwing an exception (327ms)
    
      Testing themes against JOHNNY-TROUBLE resume (FRESH)
        ✓ HELLO-WORLD theme (FRESH) should generate without throwing an exception
        ✓ COMPACT theme (FRESH) should generate without throwing an exception (1553ms)
        ✓ MODERN theme (FRESH) should generate without throwing an exception (1313ms)
        ✓ UNDERSCORE theme (FRESH) should generate without throwing an exception
    ^C

The AWESOME theme should be next, but the thread just goes to 100% CPU until I kill it manually after a few minutes.

For a minimal reproducible example:
- In `test/all.js` comment out all test scripts except `test-themes`.
- In `test/scripts/test-themes.js` edit the list of `genTheme()` calls to only:
````
    genTheme(fmt, src, 'jsonresume-theme-boilerplate', 'node_modules/jsonresume-theme-boilerplate' );
    genTheme(fmt, src, 'awesome');
````

This problem occurs with both the current HEAD of branch master (1.8.0) and the current HEAD of branch dev (1.9.0) and in both Node.js 0.10.29 on Debian Jessie and Node.js 4.6.1 on Debian unstable.

**Comment by hacksalot** [comment]
EXTREMELY helpful. Thanks for this.

**Comment by hacksalot** [comment]
Also triggers #198, strangely enough.

**Comment by hacksalot** [comment]
Should be fixed in 1.9.0-beta. Thanks
